### PR TITLE
Fixes resist cooldown

### PR DESCRIPTION
**Pull request by Garen7**
## About The Pull Request
Resist cooldown was accidentaly removed by #4356, this PR returns it.

### CODE REVIEW

**Comment by Coureere** [comment]
Please, don't add bug with unability to click anything after you unbuckle via resist verb. It was purposefully. This is as vital to combat as resist cooldown itself, but it must be implemented the way it works only for grab resisting, but doesn't botches unbuckling resisting. Either close it or fix it so it doesn't forbid you from clicking when you resisted out of buckled state.

File: code/modules/mob/living/carbon/resist.dm [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	set category = "IC"
 
 	if(!stat && can_click())
+		setClickCooldown(20)
```
Comment (ACCount12) [inline code/modules/mob/living/carbon/resist.dm]: `20` is definitely overboard. Use `DEFAULT_QUICK_COOLDOWN` (4) or at least `DEFAULT_ATTACK_COOLDOWN` (8).

**Comment by Kurgis** [comment]
The setClickCooldown compounds with the unbuckling action, have the resist macro check if they are not buckled before applying.

**Comment by Garen7** [comment]
@Kurgis set cooldown sets the cooldown, it doesn’t increment it if that’s what you mean by compounding

@Coureere not sure where you need to unbuckle from a chair midcombat but it doesn’t matter because im going to change this to fix the other bug in like a few hours when I get home

File: code/modules/mob/living/carbon/resist.dm (hunk 1 above)
Comment (Garen7) [reply code/modules/mob/living/carbon/resist.dm]: This was already tried and tested in #4113 and #4138 not long ago, are you sure you want that?

**Pull request merged**